packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -119,10 +127,61 @@ export class SerializedStateManager {
 		} else {
 			const { baseSnapshot, snapshotBlobs } = this.pendingLocalState;
 			this.snapshot = { baseSnapshot, snapshotBlobs };
+			this.refreshSnapshot ??= (async () => {
+				this.latestSnapshot = await getLatestSnapshotInfo(
+					this.mc,
+					this.storageAdapter,
+					supportGetSnapshotApi,
+				);
+				this.updateSnapshotAndProcessedOpsMaybe();
+			})();
```

Comment:
```suggestion
				this.latestSnapshot = getLatestSnapshotInfo(
					this.mc,
					this.storageAdapter,
					supportGetSnapshotApi,
				).then(()=>this.updateSnapshotAndProcessedOpsMaybe());
```
just a bit of syntax simplification

Reply:
I believe they're not entirely equivalent since latestSnapshot is no longer a SnapshotInfo object. If this is needed, then there are more things in the code that need to be changed